tests/integration/file_formats/test_main.py

```diff
@@ -132,9 +132,7 @@ def test_read_rows_validates_categories(test_file):
     # the schema but with CSV we can only validate individual values
     errors = {
         "dataset.arrow": (
-            "Unexpected categories in column 'c'\n"
-            "  Categories: A, B\n"
-            "  Expected: X, Y"
+            "Unexpected categories in column 'c'\n  Categories: A, B\n  Expected: X, Y"
```

Comment:
Ah, this is a bit of shame. I think the string is quite a lot less readable now. I wonder if we should use `textwrap.dedent` here?